File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-7.js
const prompt = require('prompt-sync')()

const estatura1 = Number(prompt('Estatura 1:') )
const estatura2 = Number(prompt('Estatura 2:') )
const estatura3 = Number(prompt('Estatura 3:') )

if(estatura1 == estatura2 || estatura1 == estatura3 || estatura2 == estatura3){
    console.log('Há, pelo menos, 2 pessoas com a mesma estatura.')
}else{
    if(estatura1 > estatura2 && estatura1 > estatura3){
        console.log(`Maior estatura:${estatura1}`)
    }else if(estatura2 > estatura3){
        console.log(`Maior estatura:${estatura2}`)
    }else{
        console.log(`Maior estatura:${estatura3}`)
    }
}

File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_7.py
estatura_1 = int(input('Digite a 1° estatura:'))
estatura_2 = int(input('Digite a 2° estatura:'))
estatura_3 = int(input('Digite a 3° estatura:'))

if estatura_1 == estatura_2 or estatura_1 == estatura_3 or estatura_2 == estatura_3:
    print('Há pelo menos 2 pessoas com a mesma estatura.')
else:
    if estatura_1 > estatura_2 and estatura_1 > estatura_3:
        print(f'Maior estatura é a 1°: {estatura_1}')
    else:
        if estatura_2 > estatura_1 and estatura_2 > estatura_3:
            print(f'Maior estatura é a 2°: {estatura_2}')
        else:
            print(f'Maior estatura é a 3°: {estatura_3}')
